### PR TITLE
Create new component Select supporting Icons within Option List (issue/2921)

### PR DESCRIPTION
**Pull request by henrijoss**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2921

#### Scope

We want to provide a fully encapsulated Select component like we've did for Multi Select or Pin Code. In addition, the Select component shall provide the possibility to visualise icons/images next to the label of an option list item.

#### Resolved Issue

Resolves [#2921](https://github.com/porsche-design-system/porsche-design-system/issues/2921)

Follow Up Issue [#3054](https://github.com/porsche-design-system/porsche-design-system/issues/3054)

### CODE REVIEW

File: packages/components-angular/src/app/examples/select-example-required.component.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'page-select-example-required',
+  template: `
+    <label>
+      <input type="checkbox" name="required" [checked]="isRequired" (change)="onChangeRequired()" />
+      Required
+    </label>
+    <br />
```
Comment (denyo) [inline packages/components-angular/src/app/examples/select-example-required.component.ts:10]: I guess `label`s should become block elements instead.

File: packages/components-js/tests/vrt/playwright/specs/common/components.vrt.ts [hunk 2]
```diff
@@ -13,17 +13,17 @@ import {
 const components = (TAG_NAMES as unknown as TagName[])
```
Comment (denyo) [inline packages/components-js/tests/vrt/playwright/specs/common/components.vrt.ts:13]: I think `form-fields.vrt.ts` should be extended, too.

File: packages/components-js/tests/vrt/playwright/specs/states/select.vrt.ts [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+import { schemes, themes, viewportWidthM } from '@porsche-design-system/shared/testing/playwright.vrt.config';
+import {
+  forceFocusHoverState,
+  forceFocusState,
+  forceHoverState,
+  getPlaygroundPseudoStatesMarkup,
+  type PrefersColorScheme,
+  setContentWithDesignSystem,
+} from '../../helpers';
+import { type Theme } from '@porsche-design-system/utilities-v2';
```
Comment (denyo) [inline packages/components-js/tests/vrt/playwright/specs/states/select.vrt.ts:11]: Would be safer/better to import from components since this is somewhat unrelated.

File: packages/components-react/projects/uxpin-wrapper/src/form/SelectWrapperDummy/SelectWrapperDummy.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { partitionProps } from '../../form-utils';
 
 export type SelectProps = PSelectWrapperProps & DummySelectProps;
 
-export const Select = (props: SelectProps): JSX.Element => {
+export const SelectWrapperDummy = (props: SelectProps): JSX.Element => {
```
Comment (denyo) [inline packages/components-react/projects/uxpin-wrapper/src/form/SelectWrapperDummy/SelectWrapperDummy.tsx:7]: Not sure if this will break existing prototypes? Perhaps an alias would be better or shouldn't this be called `SelectWrapper` since this in the end is a template?

File: packages/components-angular/src/app/examples/select-example-dynamic.component.ts [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+@Component({
+  selector: 'page-select-example-dynamic',
+  template: `
+    <label>
+      Value:
+      <input name="input-value" type="text" [value]="inputValue" (input)="onChangeInput($event)" placeholder="e.g. 1" />
+    </label>
+    <button type="button" (click)="onSetValue()">Set Value</button>
+    <button type="button" (click)="onResetValue()">Reset value</button>
```
Comment (denyo) [inline packages/components-angular/src/app/examples/select-example-dynamic.component.ts:12]: Namings across newly added (but probably also old) examples is mixed. Sometimes the handlers are called `handleSubmit`, but here it is essentially `onSubmit`.  
Consistency would be good.  
Same situation in react and I guess also vue.

File: packages/components-react/src/examples/SelectExampleDynamic.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        <input name="input-value" type="text" value={inputValue} onChange={onChangeInput} placeholder="e.g. 1,2" />
+      </label>
+      <button type="button" onClick={onSetValue}>
+        Set Value
+      </button>
+      <button type="button" onClick={onResetValue}>
+        Reset value
+      </button>
+
+      <PSelect name="options" label="Some Label" value={selectedValue} onUpdate={handleUpdate}>
```
Comment (denyo) [inline packages/components-react/src/examples/SelectExampleDynamic.tsx:55]: Here you can see the mixup quite good within a single file.

File: packages/components-vue/src/examples/SelectExample.vue [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  const lastSubmittedData = ref('none');
+  const handleSubmit = (e: Event) => {
+    const formData = new FormData(e.target as HTMLFormElement);
+    lastSubmittedData.value = formData.get('options')?.toString() || 'none';
+  };
+</script>
+
+<template>
+  <form @submit.prevent="handleSubmit">
+    <PSelect :name="'options'" :label="'Some Label'" value="'a'">
```
Comment (denyo) [inline packages/components-vue/src/examples/SelectExample.vue:14]: I guess copy/paste from multi select but here it is just singular. 
```suggestion
    <PSelect :name="'option'" :label="'Some Label'" value="'a'">
```

File: packages/components-vue/src/examples/SelectExample.vue [hunk 8]
```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+  import { PSelect, PSelectOption } from '@porsche-design-system/components-vue';
+  import { ref } from 'vue';
+
+  const lastSubmittedData = ref('none');
+  const handleSubmit = (e: Event) => {
+    const formData = new FormData(e.target as HTMLFormElement);
+    lastSubmittedData.value = formData.get('options')?.toString() || 'none';
```
Comment (denyo) [inline packages/components-vue/src/examples/SelectExample.vue:8]:
```suggestion
    lastSubmittedData.value = formData.get('option')?.toString() || 'none';
```

File: packages/components-vue/src/examples/SelectExampleDynamic.vue [hunk 9]
```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+  import { type SelectUpdateEventDetail, PSelect, PSelectOption } from '@porsche-design-system/components-vue';
+  import { ref } from 'vue';
+
+  const selectedValue = ref<string>('1');
+  const inputValue = ref('');
+  const optionCount = ref(3);
```
Comment (denyo) [inline packages/components-vue/src/examples/SelectExampleDynamic.vue:7]: Why not explicitly type them all consistently?

File: packages/components/CHANGELOG.md [hunk 10]
```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Added
 
+- `Select` ([#3008](https://github.com/porsche-design-system/porsche-design-system/pull/3008))
```
Comment (denyo) [inline packages/components/CHANGELOG.md:21]: Should go into `Unreleased`.

File: packages/components/scripts/generateDSRComponents.ts [hunk 11]
```diff
@@ -272,7 +272,7 @@ import { get${componentName}Css } from '${stylesBundleImportPath}';
         .replace(/ {\.\.\.toast}/, '') // toast
         .replace(/return this\.selectRef\.selectedIndex;/, 'return 0;') // select-wrapper-dropdown
         .replace(/determineDropdownDirection\(this\.props\,.+\)/, "'down'") // select-wrapper-dropdown
-        .replace(/getDropdownDirection\(this\.props.+\)/, "'down'") // multi-select
+        .replace(/(getDropdownDirection|getSelectDropdownDirection)\(this\.props.+\)/, "'down'") // multi-select
```
Comment (denyo) [inline packages/components/scripts/generateDSRComponents.ts:275]:
```suggestion
        .replace(/(getDropdownDirection|getSelectDropdownDirection)\(this\.props.+\)/, "'down'") // select and multi-select
```

File: packages/components/src/assets/718.png
Comment (denyo) [inline packages/components/src/assets/718.png:1]: Leftovers? Or why not reuse the assets package?

File: packages/components/src/components/multi-select/multi-select/multi-select-styles.ts [hunk 12]
```diff
@@ -61,6 +61,7 @@ export const getComponentCss = (
             position: 'absolute',
```
Comment (denyo) [inline packages/components/src/components/multi-select/multi-select/multi-select-styles.ts:61]: Now we could reuse `INTERNAL_MULTISELECT_SLOT` in the previous line.

File: packages/components/src/components/multi-select/multi-select/multi-select-utils.ts [hunk 13]
```diff
@@ -15,6 +15,8 @@ export type MultiSelectUpdateEvent = {
 };
 export type MultiSelectUpdateEventDetail = MultiSelectUpdateEvent;
 
+export const INTERNAL_MULTISELECT_SLOT = 'internal-select';
```
Comment (denyo) [inline packages/components/src/components/multi-select/multi-select/multi-select-utils.ts:18]: Underscore in the middle is missing? `INTERNAL_MULTI_SELECT_SLOT`

File: packages/components/src/components/select/select-option/select-option-styles.ts [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+  addImportantToEachRule,
+  getHighContrastColors,
+  getThemedColors,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+  prefersColorSchemeDarkMediaQuery,
+} from '../../../styles';
+import { borderRadiusSmall, fontLineHeight, spacingStaticSmall } from '@porsche-design-system/utilities-v2';
+import { Styles } from 'jss';
```
Comment (denyo) [inline packages/components/src/components/select/select-option/select-option-styles.ts:14]:
```suggestion
import type { Styles } from 'jss';
```

File: packages/components/src/components/select/select-option/select-option-styles.ts [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+
+export const getComponentCss = (theme: Theme): string => {
+  return getCss({
+    '@global': {
+      ':host': addImportantToEachRule({
```
Comment (denyo) [inline packages/components/src/components/select/select-option/select-option-styles.ts:19]: This has no `display` style so it is probably `inline` by default. Not sure if this is fine.
In case there are no other styles, it looks like everything is important and therefore both `:host` and `::slotted(img)` could be combined in a single object and passed to `addImportantToEachRule` instead of calling it twice with object composition.

File: packages/components/src/components/select/select-option/select-option-utils.ts [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+  const notValid = directChildren.some((child) => {
+    return !(
+      (child.nodeType === Node.ELEMENT_NODE && (child as HTMLElement).tagName.toLowerCase() === 'img') ||
+      child.nodeType === Node.TEXT_NODE
+    );
+  });
+
+  if (notValid) {
+    consoleError(
+      `child HTMLElements of ${getTagNameWithoutPrefix(host)} are invalid. Expected all of: text or img.`,
```
Comment (denyo) [inline packages/components/src/components/select/select-option/select-option-utils.ts:25]: Maybe refer to it as plain text or so since  there is an `img` tag but not a `text` tag.

File: packages/components/src/components/select/select-option/select-option.tsx [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+  private onClick = (): void => {
+    this.host.dispatchEvent(
+      new CustomEvent('internalOptionUpdate', {
+        bubbles: true,
+      })
+    );
+  };
+
+  private onSlotChange = (e: Event): void => {
+    validateSelectOption(e.target as HTMLSlotElement, this.host);
```
Comment (denyo) [inline packages/components/src/components/select/select-option/select-option.tsx:80]: Could type it like this instead of casting.
```suggestion
  private onSlotChange = (e: Event & { target: HTMLSlotElement }): void => {
    validateSelectOption(e.target, this.host);
```

File: packages/components/src/components/select/select/select-styles.ts [hunk 18]
```diff
@@ -0,0 +1,237 @@
...
+import {
+  borderRadiusSmall,
+  borderWidthBase,
+  fontLineHeight,
+  spacingStaticSmall,
+  spacingStaticXSmall,
+  textSmallStyle,
+} from '@porsche-design-system/utilities-v2';
+import { JssStyle } from 'jss';
+import { FormState } from '../../../utils/form/form-state';
```
Comment (denyo) [inline packages/components/src/components/select/select/select-styles.ts:30]:
```suggestion
import type { JssStyle } from 'jss';
import type { FormState } from '../../../utils/form/form-state';
```

File: packages/components/src/components/select/select/select-styles.ts [hunk 19]
```diff
@@ -0,0 +1,237 @@
+import type { BreakpointCustomizable, Theme } from '../../../types';
+import { getCss, SelectDropdownDirectionInternal } from '../../../utils';
```
Comment (denyo) [inline packages/components/src/components/select/select/select-styles.ts:2]:
```suggestion
import { getCss, type SelectDropdownDirectionInternal } from '../../../utils';
```

File: packages/components/src/components/select/select/select-styles.ts [hunk 20]
```diff
@@ -0,0 +1,237 @@
...
+  isDisabled: boolean,
+  hideLabel: BreakpointCustomizable<boolean>,
+  state: FormState,
+  isWithinForm: boolean,
+  theme: Theme
+): string => {
+  return getCss({
+    '@global': {
+      ':host': {
+        ...addImportantToEachRule({
```
Comment (denyo) [inline packages/components/src/components/select/select/select-styles.ts:49]: Useless object composition.

File: packages/components/src/utils/select/select-dropdown.ts [hunk 21]
```diff
@@ -3,7 +3,7 @@ import { OPTION_HEIGHT } from '../../styles/option-styles';
 export const SELECT_DROPDOWN_DIRECTIONS_INTERNAL = ['down', 'up'] as const;
 export type SelectDropdownDirectionInternal = (typeof SELECT_DROPDOWN_DIRECTIONS_INTERNAL)[number];
 export const SELECT_DROPDOWN_DIRECTIONS = [...SELECT_DROPDOWN_DIRECTIONS_INTERNAL, 'auto'] as const;
-export type SelectDropdownDirection = (typeof SELECT_DROPDOWN_DIRECTIONS)[number];
+export type SelectDropdownDirections = (typeof SELECT_DROPDOWN_DIRECTIONS)[number];
```
Comment (denyo) [inline packages/components/src/utils/select/select-dropdown.ts:6]: Should stay singular since when the type is used it is a single value, not an array.

File: packages/components/src/components/select/select/select.tsx [hunk 22]
```diff
@@ -0,0 +1,393 @@
...
+  private comboboxContainer: HTMLDivElement;
+  private combobox: HTMLButtonElement;
+  private listElement: HTMLDivElement;
+  private selectOptions: SelectOption[] = [];
+  private form: HTMLFormElement;
+  private isWithinForm: boolean;
+  private preventOptionUpdate = false; // Used to prevent value watcher from updating options when options are already updated
+
+  private searchString: string = '';
+  private searchTimeout: NodeJS.Timeout | number = null;
```
Comment (denyo) [inline packages/components/src/components/select/select/select.tsx:129]: NodeJS type is probably not the best one for something running in the browser.

File: packages/components/src/components/select/select/select.tsx [hunk 23]
```diff
@@ -0,0 +1,393 @@
...
+    this.updateOptions();
+    updateSelectOptions(this.selectOptions, this.value);
+    if (this.isWithinForm) {
+      this.nativeSelect = initNativeSelect(this.host, this.name, this.disabled, this.required);
+      updateNativeSelectOption(this.nativeSelect, this.selectOptions);
+    }
+  }
+
+  public componentDidLoad(): void {
+    getShadowRootHTMLElement(this.host, 'slot').addEventListener('slotchange', this.onSlotchange);
```
Comment (denyo) [inline packages/components/src/components/select/select/select.tsx:167]: Here we do it imperatively, I think in `p-select-option` it was done declaratively in JSX.  
Do both work equally now? Why not stay consistent?

File: packages/components/src/utils/select/keyboard-behavior.spec.ts [hunk 24]
```diff
@@ -0,0 +1,286 @@
...
+    Array(amount),
+    (_, idx) =>
+      ({
+        value: values?.[idx] ? values[idx] : `Value ${idx}`,
+        textContent: textContents?.[idx] ? textContents[idx] : `Option ${idx}`,
+        selected: selectedIndices.includes(idx),
+        highlighted: highlightedIndex === idx,
+        disabled: disabledIndex === idx,
+        hidden: hiddenIndex === idx,
+      }) as unknown as Option
```
Comment (denyo) [inline packages/components/src/utils/select/keyboard-behavior.spec.ts:51]: Could probably type the generic in `Array.from<T>` instead of casting.

File: packages/components/src/utils/select/keyboard-behavior.ts [hunk 25]
```diff
@@ -0,0 +1,249 @@
+import { forceUpdate } from '@stencil/core';
+import { HTMLStencilElement } from '@stencil/core/internal';
```
Comment (denyo) [inline packages/components/src/utils/select/keyboard-behavior.ts:2]:
```suggestion
import type { HTMLStencilElement } from '@stencil/core/internal';
```

File: packages/components/src/utils/select/keyboard-behavior.ts [hunk 26]
```diff
@@ -0,0 +1,249 @@
...
+
+export const selectActions = {
```
Comment (denyo) [inline packages/components/src/utils/select/keyboard-behavior.ts:4]: I guess this was the `enum` before and this is where the numeric values are coming from. What's the benefit? Or why not use define a literal type directly instead?

File: packages/components/src/utils/select/keyboard-behavior.ts [hunk 27]
```diff
@@ -0,0 +1,249 @@
...
+    element.scrollIntoView({ block: 'nearest' });
+  }
+};
+
+/**
+ * Determines the action to be taken based on a keyboard event and the state of the select menu.
+ *
+ * @param {KeyboardEvent} event - The keyboard event triggering the action.
+ * @param {boolean} menuOpen - A boolean indicating whether the select menu is open or closed.
+ * @returns {selectActions} - The corresponding action to be performed.
```
Comment (denyo) [inline packages/components/src/utils/select/keyboard-behavior.ts:51]:
```suggestion
 * @returns {SelectAction} - The corresponding action to be performed.
```

File: packages/components/src/utils/select/keyboard-behavior.ts [hunk 28]
```diff
@@ -0,0 +1,249 @@
...
+    }
+  }
+};
+
+/**
+ * Gets the updated index based on the current index, maximum index, and the select action.
+ *
+ * @param {number} currentIndex - The current index in the list of options.
+ * @param {number} maxIndex - The maximum index in the list of options.
+ * @param {selectActions} action - The select action indicating how to update the index.
```
Comment (denyo) [inline packages/components/src/utils/select/keyboard-behavior.ts:99]: Looks like several jsdocs weren't replaced/refactored.
```suggestion
 * @param {SelectAction} action - The select action indicating how to update the index.
```

File: packages/components/stencil.config.ts [hunk 29]
```diff
@@ -27,6 +27,7 @@ export const bundles: { components: TagName[] }[] = [
   { components: ['p-segmented-control', 'p-segmented-control-item'] },
   { components: ['p-select-wrapper', 'p-select-wrapper-dropdown'] },
   { components: ['p-multi-select', 'p-multi-select-option'] },
+  { components: ['p-select', 'p-select-option'] },
```
Comment (denyo) [inline packages/components/stencil.config.ts:30]: Could be ordered alphabetically.

File: packages/storefront/src/assets/718.png
Comment (denyo) [inline packages/storefront/src/assets/718.png:1]: Why not use assets package or even model signature component or how are consumers supposed to rebuild this?

File: packages/components-react/projects/uxpin-wrapper/src/form/SelectWrapperDummy/SelectWrapperDummy.tsx (hunk 4 above)
Comment (henrijoss) [reply packages/components-react/projects/uxpin-wrapper/src/form/SelectWrapperDummy/SelectWrapperDummy.tsx:7]: Theres already a `SelectWrapper` component generated as well. This can't be named Select since the `Select` will also be generated.

File: packages/components-vue/src/examples/SelectExample.vue (hunk 7 above)
Comment (henrijoss) [reply packages/components-vue/src/examples/SelectExample.vue:14]: I guess `options` is also fine for a select name where you have multiple options to choose from

File: packages/components-angular/src/app/examples/select-example-controlled.component.ts [hunk 30]
```diff
@@ -0,0 +1,30 @@
...
+  template: `
+    <p-select name="options" label="Some Label" [value]="selectedValue" (update)="onUpdate($event)">
+      <p-select-option value="a">Option A</p-select-option>
+      <p-select-option value="b">Option B</p-select-option>
+      <p-select-option value="c">Option C</p-select-option>
+      <p-select-option value="d">Option D</p-select-option>
+      <p-select-option value="e">Option E</p-select-option>
+      <p-select-option value="f">Option F</p-select-option>
+    </p-select>
+    <button type="submit">Submit</button>
```
Comment (denyo) [inline packages/components-angular/src/app/examples/select-example-controlled.component.ts:15]: Looks like wrapping `form` is missing or this button can be removed.

File: packages/components-js/tests/e2e/puppeteer/specs/select.e2e.ts [hunk 31]
```diff
@@ -0,0 +1,1461 @@
+import type { Page } from 'puppeteer';
+import type { Components } from '@porsche-design-system/components';
```
Comment (denyo) [inline packages/components-js/tests/e2e/puppeteer/specs/select.e2e.ts:2]: Can we import this from src? Otherwise the import can't be found with `components:start`.

File: packages/components-js/tests/vrt/playwright/specs/__screenshots__/form-fields-1920-states-theme-light-chrome.png
Comment (denyo) [inline packages/components-js/tests/vrt/playwright/specs/__screenshots__/form-fields-1920-states-theme-light-chrome.png:1]: Hard to tell which component this is but the border color is different for `readonly`.

Bildschirmfoto 2024-02-26 um 09 29 57

File: packages/components/src/components/select/select/select-styles.ts [hunk 32]
```diff
@@ -0,0 +1,239 @@
...
+  const { formStateColor, formStateHoverColor } = getThemedFormStateColors(theme, state);
+  const { formStateColor: formStateColorDark, formStateHoverColor: formStateHoverColorDark } = getThemedFormStateColors(
+    'dark',
+    state
+  );
+
+  return {
+    textAlign: 'start', // TODO: Newly added
+    overflowX: 'hidden', // TODO: Newly added
+    whiteSpace: 'nowrap', // TODO: Newly added
```
Comment (denyo) [inline packages/components/src/components/select/select/select-styles.ts:123]: Not sure if I would know what todo here :)

File: packages/components/src/components/select/select/select.tsx [hunk 33]
```diff
@@ -0,0 +1,391 @@
...
+    }
+    // Necessary to update selected options in placeholder
+    forceUpdate(this.host);
+  };
+
+  private updateOptions = (): void => {
+    this.selectOptions = Array.from(this.host.children).filter(
+      (el) => el.tagName !== 'SELECT' && el.slot !== 'label' && el.slot !== 'description' && el.slot !== 'message'
+    ) as HTMLPMultiSelectOptionElement[];
+    this.selectOptions.forEach((child) => throwIfElementIsNotOfKind(this.host, child, 'p-select-option'));
```
Comment (denyo) [inline packages/components/src/components/select/select/select.tsx:275]: Hmm, quite a round trip with n extra cycles when validating like this.  
Instead, prefixed `p-select-option`s could be selected directly and if the remaining children don't match the named slot and `SELECT` condition from above then something is wrong and we should trow.

File: packages/components/src/components/select/select/select-utils.ts [hunk 34]
```diff
@@ -0,0 +1,137 @@
...
+  determineDropdownDirection,
+  getHighlightedSelectOptionIndex,
+  getUsableSelectOptions,
+  type SelectDropdownDirectionInternal,
+  type SelectComponentsDropdownDirection,
+  setAttribute,
+  setAttributes,
+  type Theme,
+} from '../../../utils';
+import { SelectOptionInternalHTMLProps } from '../select-option/select-option-utils';
```
Comment (denyo) [inline packages/components/src/components/select/select/select-utils.ts:13]:
```suggestion
import type { SelectOptionInternalHTMLProps } from '../select-option/select-option-utils';
```

File: packages/components/src/components/select/select/select.tsx [hunk 35]
```diff
@@ -0,0 +1,391 @@
...
+    if (this.isOpen === open) {
+      return;
+    }
+    this.isOpen = open;
+  };
+
+  private onComboKeyDown = (event: KeyboardEvent): void => {
+    const { key } = event;
+
+    const action = getActionFromKey(event, this.isOpen);
```
Comment (denyo) [inline packages/components/src/components/select/select/select.tsx:305]: Perhaps rename to `getActionFromKeyboardEvent` since this is how you call function based on parameters. Currently it is a bit awkward that `key` is destructured beforehand but then not passed to this function.

File: packages/components-js/tests/vrt/playwright/specs/__screenshots__/form-fields-1920-states-theme-light-chrome.png
Comment (henrijoss) [reply packages/components-js/tests/vrt/playwright/specs/__screenshots__/form-fields-1920-states-theme-light-chrome.png:1]: Added to the improvements [issue](https://github.com/orgs/porsche-design-system/projects/14/views/1?pane=issue&itemId=53839901) since it will be solved when using the shared styles

**Review by stephanschroeter — APPROVED** [review]
